client/src/components/QueryComponent/QueryComponent.js

```diff
@@ -170,10 +203,21 @@ class QueryComponent extends Component {
 
   render() {
     const { queries, selectedQuery, selectedQueryKey, showSaveQueryModal, newQueryName,
-      showConfirmDeleteModal, showConfirmOverwriteModal, isOverlayResultsChecked } = this.state;
+      showConfirmDeleteModal, showConfirmOverwriteModal, isOverlayResultsChecked, rowCount, display, selectedQueryMultiline } = this.state;
 
     return (
       <>
+        {display && <div className="qc-monaco-layer" onBlur={this.handleEditorBlur} >
+          <Editor
+            height={rowCount}
+            theme="vs-dark"
+            language="sql"
```

Comment:
IIRC the ADT query language isn't strictly ANSI SQL.

Is this extensible by any chance?

Reply:
We have another requirement to make a customize language